From: "Diego M. Rodriguez" <diego@moreda.io>
Date: Fri, 24 Sep 2021 12:51:25 +0200
Subject: Exclude pythran/pythonic/patch/README.rst

Exclude the "pythran/pythonic/patch/README.rst" file in the final
distribution, in order to comply with the lintian warning
package-contains-documentation-outside-usr-share-doc.
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 74d3c81..59d68ec 100644
--- a/setup.py
+++ b/setup.py
@@ -124,7 +124,7 @@ class DevelopWithThirdParty(develop, BuildWithThirdParty):
 # Cannot use glob here, as the files may not be generated yet
 boost_headers = ['boost/' + '*/' * i + '*.hpp' for i in range(1, 20)]
 xsimd_headers = ['xsimd/' + '*/' * i + '*.hpp' for i in range(1, 20)]
-pythonic_headers = ['*/' * i + '*.hpp' for i in range(9)] + ['patch/*']
+pythonic_headers = ['*/' * i + '*.hpp' for i in range(9)] + ['patch/complex']
 
 
 # read longdescr from README
